### PR TITLE
An Introduction to JavaScript

### PR DESCRIPTION
**Pull request by longo-andrea**
Reviewed An Introduction to JavaScript article. I have already make a grammar check, only a review is needed.

### CODE REVIEW

**Comment by longo-andrea** [comment]
> Error: the article already translated in the Progress Issue #1 ⁉️

Just a review of an already translated article

File: 1-js/01-getting-started/1-intro/article.md [hunk 1]
```diff
@@ -99,15 +99,15 @@ Quando si ha in programma di imparare una nuova tecnologia, è fondamentale veri
 
 La sintassi di JavaScript non soddisfa le necessità di tutti. Alcune persone necessitano di caratteristiche differenti.
 
-Questo è prevedibile, poichè i progetti e i requisiti sono diversi di persona in persona.
+Questo è prevedibile, poiché i progetti e i requisiti sono diversi di persona in persona.
 
-Quindi recentemente un'elevata quantità di nuovi linguaggi è apparsa, che vengono *convertiti* in JavaScript prima di essere eseguite nel browser.
+Quindi recentemente un'elevata quantità di nuovi linguaggi è apparsa, che vengono *convertiti* in JavaScript prima di essere eseguiti nel browser.
```
Comment (mean2me) [inline 1-js/01-getting-started/1-intro/article.md:104]: I'd say: "Recentemente, per questo motivo, sono nati molti nuovi linguaggi che vengono convertiti i JavaScript, prima di essere eseguiti nel browser."

File: 1-js/01-getting-started/1-intro/article.md [hunk 2]
```diff
@@ -99,15 +99,15 @@ Quando si ha in programma di imparare una nuova tecnologia, è fondamentale veri
...
 
 Gli strumenti moderni rendono la conversione molto veloce e pulita, consentendo agli sviluppatori di programmare in un altro linguaggio e di auto-convertirlo "sotto il cofano".
 
 Esempi di alcuni linguaggi:
 
-- [CoffeeScript](http://coffeescript.org/) è un linguaggio con sintassi "leggera" per JavaScript, introduce una sintassi più breve, consente di scirvere codice più pulito e preciso. Amato dagli sviluppatori provenienti da Ruby.
+- [CoffeeScript](http://coffeescript.org/) è un linguaggio con sintassi "leggera" per JavaScript, introduce una sintassi più breve, consente di scrivere codice più pulito e preciso. Amato dagli sviluppatori provenienti da Ruby.
```
Comment (mean2me) [inline 1-js/01-getting-started/1-intro/article.md:110]: I'd propose: "introduce una sintassi semplificata che consente di scrivere codice più leggibile".
It doesn't change the meaning but it's closer to what an italian would write. 
IMHO

**Review by mean2me — APPROVED** [review]
I just propose minor changes to make it more readable and smooth for italians. It'd be good enough as is anyway.

**Comment by longo-andrea** [comment]
> I just propose minor changes to make it more readable and smooth for italians. It'd be good enough as is anyway.

Yes, those two paragraphs have been translated too literally, thanks for review! I already made the changes

**Pull request merged**